src/metadata/metadata_superblock.c

```diff
@@ -723,7 +789,7 @@ struct ocf_pipeline_properties ocf_metadata_sb_crc_recovery_pipeline_props = {
 	.priv_size = sizeof(struct ocf_metadata_context),
 	.finish = ocf_metadata_sb_crc_recovery_finish,
 	.steps = {
-		OCF_PL_STEP(ocf_metadata_check_crc_sb_config),
+		//OCF_PL_STEP(ocf_metadata_check_crc_sb_config),
```

Comment:
I'm completely unaware of this pipeline existence. I think it is not used anywhere? The only reference is in function ocf_metadata_sb_crc_recovery(), which doesn't appear to have any callers and is not in OCF API.

Reply:
... however I think the old implementation is correct, you should not remove the first step